### PR TITLE
* migrate to ns6

### PR DESCRIPTION
**Pull request by jcarolus**
Takes care of #12 

As known, not backward compatible - Under Reguirements a notice has been added in the README but I leave the version up to you @lochstar .

### CODE REVIEW

**Comment by jcarolus** [comment]
@lochstar hope everything is ok. Can you have another look at this?

File: src/package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "typings": "cast.d.ts",
   "nativescript": {
     "platforms": {
-      "android": "5.4.0",
-      "ios": "5.4.2"
+      "android": "5.3.1",
+      "ios": "5.3.1"
```
Comment (DerekTBrown) [inline src/package.json]: Why were these not updated to 6.0.1?

**Comment by lochstar** [comment]
Everythings good, @jcarolus. Cheers. I'll get this merged this week.

File: src/package.json (hunk 1 above)
Comment (jcarolus) [reply src/package.json]: good find. Will update it. 
Ran into a crash (testing on iOS 13) which may be related to latest xcode, so trying to figure that out as well - probably we have to upgrade the GoogleCast.framework (it's recommended for iOS 13).

Comment (jcarolus) [reply src/package.json]: updated.
Also upgrading to GoogleCast 4.4.5. Dynamic cast API for iOS fixed the crash on iOS 13 device.

**Comment by jcarolus** [comment]
@lochstar since iOS 13, Guest mode for Chromecast requires extra permissions. There are two iOS API libs introduced for this reason, one with, one without the Guest mode - the upgraded library has the guest mode. This means that any project using this plugin requires this bluetooth permission. As far as I can tell, there is no way to configure a Nativescript plugin which lib to use if we were to include both somehow.
More info on the dev page: https://developers.google.com/cast/docs/ios_sender#manual_setup

**Pull request merged**